library/build.gradle

```diff
@@ -36,4 +36,4 @@ dependencies {
 }
 
 // this script was used to upload files to bintray.
-apply from: 'bintray.gradle'
+apply from: 'https://raw.githubusercontent.com/msdx/gradle-publish/master/bintray.gradle'
```

Comment:
I'd rather not have this apply from a remote URL. Why not keep the checked-in version or an officially published Gradle plugin?